packages/p2p/src/components/buy-sell/buy-sell-table-content.jsx

```diff
@@ -10,29 +10,24 @@ import { RowComponent, BuySellRowLoader } from './row.jsx';
 import { BuySellTable } from './buy-sell-table.jsx';
 
 const BuySellTableContent = ({ is_buy, setSelectedAd }) => {
-    let item_offset = 0;
     const { list_item_limit } = useContext(Dp2pContext);
-    const [is_mounted, setIsMounted] = useState(false);
+    const mounted = useRef(false);
```

Comment:
any reason why we dont use state?

Reply:
We don't use state because the value wouldn't persist nor could it be updated after the component is unmounted. The unmounted component setState error happened because we were trying to set the state of `is_mounted` to `false` when the component was already unmounted.